docs/charts/doughnut.md

```diff
@@ -63,6 +63,7 @@ The doughnut/pie chart allows a number of properties to be specified for each da
 | [`hoverBackgroundColor`](#interations) | [`Color`](../general/colors.md) | Yes | Yes | `undefined`
 | [`hoverBorderColor`](#interactions) | [`Color`](../general/colors.md) | Yes | Yes | `undefined`
 | [`hoverBorderWidth`](#interactions) | `Number` | Yes | Yes | `undefined`
+| [`weight`](#styling) | `Number` | No | No | `1`
```

Comment:
To be consistent, this should be 
```
| [`weight`](#styling) | `Number` | - | - | `1`
```

Reply:
I'll look to include this change in the next update.  I would like to hear from you about the `weight: 0` question.